tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -791,4 +793,33 @@ public function testGetListSequencesSQL(): void
             $this->platform->getListSequencesSQL('test_db'),
         );
     }
+
+    public function testAlterTableChangeJsonToJsonb(): void
+    {
+        $table = new Table('mytable');
+        $table->addColumn('payload', Types::JSON);
+
+        $tableDiff = new TableDiff($table, changedColumns: [
+            'payload' => new ColumnDiff(
+                $table->getColumn('payload'),
+                (new Column(
+                    'payload',
+                    Type::getType(Types::JSON),
+                ))->setPlatformOption('jsonb', true),
+            ),
+        ]);
+
+        self::assertSame(
+            $this->getAlterTableAddJsonbPlatformOptionSQL(),
```

Comment:
Let's inline that method.

Reply:
Had it first inlined, then saw it was in all other cases also extracted. Can inline again